redfish-core/lib/license_service.hpp

```diff
@@ -243,6 +286,11 @@ inline void requestRoutesLicenseEntryCollection(App& app)
                                     messages::internalError(asyncResp->res);
                                     return;
                                 }
+                                if (*status == "com.ibm.License.LicenseManager."
+                                               "Status.Pending")
+                                {
+                                    return;
```

Comment:
So with this we are sending the empty response to the client, if the same license ? Which response code client is getting in this case

Reply:
here we don't any response to client, this piece of code hit when bmcweb find a property change signal on LicenseActivationStatus.
this change is ignore D-bus signal if status value is Pending

Reply:
I think we need a comment here.. This also sounds pretty hacky

Reply:
I think we need a comment here.. This also sounds pretty hacky

Reply:
Can the comment say “why” we are doing this